pkg/cmd/common.go

```diff
@@ -56,8 +56,10 @@ func addCostOptionsFlags(cmd *cobra.Command, options *CostOptions) {
 
 func addQueryBackendOptionsFlags(cmd *cobra.Command, options *query.QueryBackendOptions) {
 	cmd.Flags().StringVar(&options.ServiceName, "service-name", "kubecost-cost-analyzer", "The name of the kubecost cost analyzer service. Change if you're running a non-standard deployment, like the staging helm chart.")
+	cmd.Flags().IntVar(&options.ServicePort, "service-port", 9090, "The port of the service at which the APIs are running. If using OpenCost, you may want to set this to 9003.")
 	cmd.Flags().BoolVar(&options.UseProxy, "use-proxy", false, "Instead of temporarily port-forwarding, proxy a request to Kubecost through the Kubernetes API server.")
 	cmd.Flags().StringVarP(&options.KubecostNamespace, "kubecost-namespace", "N", "kubecost", "The namespace that kubecost is deployed in. Requests to the API will be directed to this namespace.")
+	cmd.Flags().StringVar(&options.AllocationPath, "allocation-path", "/model/allocation", "URL path at which Allocation queries can be served from the configured service. If using OpenCost, you may want to set this to '/allocation/compute'")
```

Comment:
This will definitely work, but is (of course) not the nicest UX. Have we exhausted our options for getting these to match between OS and CS?

Ever since we restructured the cmd entry-points, I think we've been able to add endpoints in the OS that only exist there, right? (Before we had to give weird endpoint names to avoid router collisions, which panic.) So can we add /model/allocation to the OS (same behavior as existing /model/allocation/compute) and unify the endpoint?